### ISSUE TITLE
NTR: Treponemal antibody titer high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8482]
New term request Treponemal antibody titer high in CSF (nadea3_181018021552): TPHA; Treponema pallidum hemoagglutination assay; TPPA; Treponema pallidum particle agglutination test; Fluorescent treponemal antibody absorption (FTA-ABS); syphilis  UMLS: 3840314

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000767
- label: Positive CSF treponemal antibody positivity
- parents: [Positive CSF pathogen-specific antibody positivity (HP:0430139)](https://hpo.jax.org/app/browse/term/HP:0430139)
- pmid: 
- def: The presence of antibodies in the cerebrospinal fluid (CSF) that react against a component of treponema pallidum (syphilis).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8482
- synonyms: Positive Treponema pallidum hemoagglutination assay(NONE)|Positive Treponema pallidum particle agglutination test(NONE)|Positive Fluorescent treponemal antibody absorption(NONE)

**Comment by kanems** [comment]
Just checking if this name actually makes sense:
" Positive CSF treponemal antibody positivity"
Should it be " Positive CSF treponemal antibody test" or 
 "CSF treponemal antibody positivity"

I don't think the double positive changes the meaning, but it reads weird.

**Comment by MickeySegal** [comment]
A philosophy professor once remarked that there was no example of a double positive being interpreted as a negative.  Robert Nozick, later one of the leading philosophers, but then young, was in the audience, and replied "yeah, yeah".

**Comment by pnrobinson** [comment]
reopening, let's fix the label, at the risk of making the world have one less reason to smile

**Comment by pnrobinson** [comment]
Fixed to "CSF treponemal antibody positivity", thanks!
